src/Okta.Sdk/Internal/CastingListAdapter{T}.cs

```diff
@@ -54,6 +54,8 @@ private T Cast(object item)
             // Fall back to primitive conversion
             try
             {
+                if (!(item is IConvertible)) { return (T)item; }
```

Comment:
Disregard this line.

Reply:
Is this part of another PR?

Reply:
No, this was an attempt to handle the exception `the object doesn't implement the `IConvertible` interface`